BitwardenShared/Core/Vault/Services/VaultTimeoutServiceTests.swift

```diff
@@ -7,12 +9,14 @@ import XCTest
 @testable import BitwardenShared
 
 @MainActor
-final class VaultTimeoutServiceTests: BitwardenTestCase {
+final class VaultTimeoutServiceTests: BitwardenTestCase { // swiftlint:disable:this type_body_length
```

Comment:
⛏️ Add tests for when all modified functions throw in their different scenarios.